### ISSUE TITLE
A feature to add specific words that aren't effected by video muting to be text only filtering

### ISSUE THREAD

**Issue by CreeperPookie** [richardfrost/AdvancedProfanityFilter#198]
Hello! For certain words (non-swear words), or a variant spelling of words I prefer, sometimes I want a word (for example, misspellings of words and I want to be replaced with the correctly spelled version, e.g. "wouldnt" for "wouldn't", "cant" for "can't", or "im" for "I'm" but that I wouldn't want it actually mute the video for). I did find a workaround, but it basically required me to install two separate versions of APF, which was quite hard to do, but would conflict with the main one.

**Comment by DarthFlappy** [comment]
I doubt that is what this add-on is for..... but if it is, you could add UK vs US spelling word like center to your list.

**Comment by richardfrost** [comment]
This has now been added to version [2.7.0](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/2.7.0). To use this feature, setup a wordlist for audio muting. To learn more about wordlists, please see [the wiki](https://github.com/richardfrost/AdvancedProfanityFilter/wiki#wordlists). Thanks!

_2.7.0 has been submitted to all the browser stores, but each must be approved before it is available. Firefox has already been approved._

**Comment by richardfrost** [comment]
This is now released and should be added. Feel free to re-open or create a new issue if it doesn't work.
